coralproject/talk
# show comments on the UserDetail drawer

## What does this PR do?

Adds the ability to see the 10 most recent comments in the UserDetail drawer. You can also moderate from this same place.

## TODO
multi-select and bulk operations

## How do I test this PR?

- click a username from the moderation queue
- you should see the UserDetail drawer open and the last 10 comments will be shown
